lang/semgrep-grammars/src/semgrep-go/test/corpus/semgrep.txt

```diff
@@ -123,11 +123,12 @@ foo(..., 5)
 --------------------------------------------------------------------------------
 
 (source_file
-  (call_expression
-    (identifier)
-    (argument_list
-      (semgrep_ellipsis)
-      (int_literal))))
+  (expression_statement
+    (call_expression
+      (identifier)
+      (argument_list
+        (semgrep_ellipsis)
+        (int_literal)))))
```

Comment:
However this should be expected, as the function call `foo(...,5)` is an expression in a statement position